src/core/lang_source.ml

```diff
@@ -542,6 +542,7 @@ let iter_sources ?on_reference ~static_analysis_failed f v =
             List.iter (function _, _, Some v -> iter_value v | _ -> ()) proto
         | FFI (proto, _) ->
             List.iter (function _, _, Some v -> iter_value v | _ -> ()) proto
```

Comment:
I think we should simply call `on_imprecise` when reaching a `FFI`. We can't check the function's body in this case so this makes sense and will suit our case here.